packages/node-bridge/bridge.js

```diff
@@ -17,21 +17,46 @@ process.on('unhandledRejection', err => {
  */
 function normalizeProxyEvent(event) {
   let bodyBuffer;
-  const { method, path, headers, encoding, body } = JSON.parse(event.body);
+  const { method, path, headers, encoding, body, payloads } = JSON.parse(
+    event.body
+  );
 
-  if (body) {
-    if (encoding === 'base64') {
-      bodyBuffer = Buffer.from(body, encoding);
-    } else if (encoding === undefined) {
-      bodyBuffer = Buffer.from(body);
+  /**
+   *
+   * @param {string} b
+   * @returns Buffer
+   */
+  const normalizeBody = b => {
+    if (b) {
+      if (encoding === 'base64') {
+        bodyBuffer = Buffer.from(b, encoding);
+      } else if (encoding === undefined) {
+        bodyBuffer = Buffer.from(b);
+      } else {
+        throw new Error(`Unsupported encoding: ${encoding}`);
+      }
     } else {
-      throw new Error(`Unsupported encoding: ${encoding}`);
+      bodyBuffer = Buffer.alloc(0);
     }
-  } else {
-    bodyBuffer = Buffer.alloc(0);
+    return bodyBuffer;
+  };
+
+  if (payloads) {
+    // @ts-ignore
```

Comment:
Should we check the type of `payloads` so we don't need `ts-ignore`?

Reply:
Updated in https://github.com/vercel/vercel/pull/7507/commits/4b911762d1148599d80a25942315b6ced288ec71